ivy/stateful/initializers.py

```diff
@@ -414,7 +415,7 @@ def create_variables(
             ivy.random_normal(
                 mean=self._mean,
                 std=self._stddev,
-                shape=self._shape,
+                shape=self._shape if not var_shape else var_shape,
```

Comment:
Just to be consistent with the other initializers, we could actually just use `var_shape` and not have `shape` in the constructor at all. Or otherwise we could also add `var_shape` to the constructors of all the other intiializers, happy to know what you think 😄

Reply:
I think the first option is simpler and is just fine. I did this for backward compatibility to not break any code in the `models` repo that might be using the initializer but I just checked now and there's none so it's fine.

Reply:
Yeah that's a good point!
@hello-fri-end given that you had added this [initializer](https://github.com/unifyai/ivy/commit/e688e9832ed1947f47b4a0000d6f4640b7a80b81), could you please confirm if there's some other part of this repository or other repository where it's being used? If not, we can change it of course 😄

Reply:
Hey @kgmann @vedpatwardhan ! I had actually added this initializer for the YOLOv7 model but ended up using something else in its place. I agree that it makes more sense not to have the `shape` argument in the constructor. Feel free to make this change :+1: